clicore/constants/constants.go

```diff
@@ -0,0 +1,5 @@
+package constants
```

Comment:
I'm wondering why this is in an entirely different package. I get that it is a constant but it's completely tied to the clicore main package and if I were a user of this package I would definitely look there first

Reply:
@EItanya there was an import cycle between clicore and contextutils